.cm/gitstream.cm

```diff
@@ -0,0 +1,81 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+
+automations:
+  # Enforces a certain format on PR titles
+  enforce_pr_title:
+    if:
+      - {{ pr.title | match(regex=titlePolicy.titleRegex) | nope }}
+    run:
+      - action: request-changes@v1
+        args:
+          comment: |
+            All PRs must be titled according to our semantic naming policy: `[Area]: <short summary>`
+
+            Area must be one of the following:
+             * DataGrid
+             * Pickers 
+             * Charts
+             * TreeView
+             * Chore
```

Comment:
We also use "docs" and "core" in lowercase.

Reply:
I would be in favor of going all lowercase for consistency. Also feels a bit less noisy to me. `smashedlowercase` seems ok, we don't have complex/long names.

Reply:
`[datagrid]` would be better. There is a component `grid` in the core so it might create confusion.

Reply:
No hard preference, but to me, for a specific React component, like `[DataGrid]` or `[TreeView]` using the capitalized names make more sense, as it would be inline with the same [convention](https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use) React uses for component names, but when it refers to a more abstract set of components like `[pickers]`, using non-capitalized seem more logical.

Reply:
I have tried to document the convention in https://www.notion.so/mui-org/GitHub-PRs-7112d03a6c4346168090b29a970c0154?pvs=4#258c9e963dc445348562a52a62052941.